### PR TITLE
Community Call 20230223 Minutes

### PR DESCRIPTION
**Pull request by horazont**

### CODE REVIEW

File: community-call/20230223.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+- Refinement and progress discussion of ongoing things
+- Focus on upcoming R4
+- Flavor naming concept conflict (OpenShift putting flavors into annotations)
+- Include nameserver in OSISM
+- Magnum Cluster-API based driver
+
+#### Overflow slot appointment
+
+- Discussed open issues for the upcoming release
+- Will have to make sure that a future tender can work on the openstack image manager
```
Comment (fkr) [inline community-call/20230223.md:22]: This is not correct and was likely not told to be understood clearly.

There are two tenders that are already running (VP01 and VP06d) where work regarding the OpenStack Image Manager is suiteable. The info was that we will make sure the work is being done as part of those.

**Review by fkr — CHANGES_REQUESTED** [review]
thanks for the minutes @horazont  - please correct the note about OpenStack Image Manager.

File: community-call/20230223.md (hunk 1 above)
Comment (horazont) [reply community-call/20230223.md:22]: Fixed hopefully, please verify!

Comment (fkr) [reply community-call/20230223.md]: I adjusted it, apparently I was not clear ;)

**Review by fkr — APPROVED** [review]
Thanks @horazont - always awesome that you do the community call minutes

**Pull request merged**